src/App.js

```diff
@@ -28,5 +28,34 @@ Feel free to have fun and experiment with others too!
 If you get stuck or need help, please reach out on Slack.
 */
 export default function App() {
-  return <div>Replace this with your beautiful JSX</div>;
+  const members = team;
+
+  return (
+    <>
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <h1>Redi React Fall 2020 Team</h1>
+      </header>
+      <table className="members-table">
+        <thead>
+          <tr>
+            <th>Photo</th>
+            <th>Name</th>
+            <th>Role</th>
+          </tr>
+        </thead>
+        <tbody>
+          {members.map((member) => (
+            <tr className="member-row">
+              <td className="avatar-cell" key={member.name}>
+                <img alt="userImage" src={member.image} />
```

Comment:
Because the `alt` attribute [primarily helps people who use screen readers to access websites](https://www.deque.com/blog/great-alt-text-introduction/#:~:text=With%20alt%20text%2C%20they%20have,the%20image%20will%20display%20instead.), we want to make sure that the text describes each individual image in the map.

One way to do this would be to use the name: 
```javascript
// On its own
alt={member.name} 

// Inside a template literal
alt={`Headshot of ${member.name}`}
```